README.md

```diff
@@ -65,72 +67,35 @@ On FreeBSD and macOS, xattrs are enabled in the default kernel.
-
-Then finally install the Nuxeo Drive requirements and Nuxeo Drive itself.
+Let's say you have installed Qt4 (from official repository or compiled manually), then install Nuxeo Drive requirements and Nuxeo Drive itself.
 These are common installation actions, not depending on the package manager:
 
-	# For Drive < 2.2.227:
-	DRIVE_VERSION=release-2.1.113
-	pip install -r https://raw.github.com/nuxeo/nuxeo-drive/$DRIVE_VERSION/requirements.txt
-	pip install -r https://raw.github.com/nuxeo/nuxeo-drive/$DRIVE_VERSION/unix-requirements.txt
-	pip install git+https://github.com/nuxeo/nuxeo-drive.git@$DRIVE_VERSION
-
-	# For Drive >= 2.2.227:
-	DRIVE_VERSION=release-2.2.323
-	pip install -r https://raw.github.com/nuxeo/nuxeo-drive/$DRIVE_VERSION/requirements.txt
-	pip install -r https://raw.github.com/nuxeo/nuxeo-drive/$DRIVE_VERSION/requirements-unix.txt
-	pip install git+https://github.com/nuxeo/nuxeo-drive.git@$DRIVE_VERSION
-
-
-### macOS Desktop Client
-
-Once the Marketplace package is installed, the macOS desktop client package can be downloaded from the **Home** > **Nuxeo Drive** tab.
-
-You can also fetch the latest development version for macOS from the [our Continous Integration server](https://qa.nuxeo.org/jenkins/view/Drive/job/Drive/job/Drive-packages/).
-
-### Windows desktop client
+    # For Drive < 2.2.227:
+    DRIVE_VERSION=release-2.1.113
+    pip install -r https://raw.github.com/nuxeo/nuxeo-drive/$DRIVE_VERSION/requirements.txt
+    pip install -r https://raw.github.com/nuxeo/nuxeo-drive/$DRIVE_VERSION/unix-requirements.txt
+    pip install git+https://github.com/nuxeo/nuxeo-drive.git@$DRIVE_VERSION
 
-Once the Marketplace package is installed, the Windows desktop client package can be downloaded from the **Home** > **Nuxeo Drive** tab.
+    # For Drive >= 2.2.227:
+    DRIVE_VERSION=release-2.2.323
+    pip install -r https://raw.github.com/nuxeo/nuxeo-drive/$DRIVE_VERSION/requirements.txt
+    pip install -r https://raw.github.com/nuxeo/nuxeo-drive/$DRIVE_VERSION/requirements-unix.txt
+    pip install git+https://github.com/nuxeo/nuxeo-drive.git@$DRIVE_VERSION
 
-You can also fetch the latest development version for Windows from [our Continuous Integration server](https://qa.nuxeo.org/jenkins/view/Drive/job/Drive/job/Drive-packages/).
 
-Once you installed the package (Administrator rights required) the new folder holding the ndrive.exe and ndrivew.exe programs will be added to your `Path` environment variable automatically.
+### macOS and Windows Desktop Clients
 
-You can start the Nuxeo Drive program from the "Start..." menu.
+Once the Marketplace package is installed, the macOS/Windows desktop client package can be downloaded from the **Home** > **Nuxeo Drive** tab.
 
-All the necessary dependencies (such as the Python interpreter and the QT / PyQt for the client side user interface) are included in this folder and should not impact any alternative version you may have already installed on your computer.
```

Comment:
Maybe interesting to keep as customers often ask if Drive requires Python or if it embeds its own version and dependencies. And also the fact that it won't interfere with any local installation, which is the purpose of the freeze.